src/GitUtilities.js

```diff
@@ -42,7 +42,7 @@ export default class GitUtilities {
 
   static commit(message, opts) {
     log.silly("commit", message);
-    const args = ["commit"];
+    const args = ["commit", "--no-gpg-sign"];
```

Comment:
I'm a little leery of adding this flag to _all_ executions of `git commit`. I have no idea how common GPG signing of commits is, although I definitely appreciate your reasoning.

I'm okay with taking this now, but we'll have to note this very well in the changelog and be open to the possibility that we may need to revert it (or put it behind an option) in the future.